src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+
+    margin: 0;
+    margin-left: 4px;
+
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &--1 :first-child,
+  &--2 :nth-child(-n+2),
+  &--3 :nth-child(-n+3),
+  &--4 :nth-child(-n+4),
+  &--5 :nth-child(n) {
+    background-image: url(/images/star-active.svg);
+  }
+}
```

Comment:
try to use cycle for
more detail in [video](https://mate.academy/learn/css-advanced/sass#/video/926)

Reply:
Did it